.github/workflows/cmake.yml

```diff
@@ -75,3 +75,101 @@ jobs:
       run: ctest
+
+  macos-build:
+    name: Xcode ${{ matrix.xcode }} on ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        xcode: [ '11.3', '11.7', '12.4' ]
+        os: [ macos-10.15 ]
+        include:
+          - xcode: '12.5'
+            os: macos-11
+          - xcode: '13.2'
+            os: macos-11
+    runs-on: ${{ matrix.os }}
+    env:
+      NODE_LABELS: OSX
+      WAF_JOBS: 3
+    steps:
+      - name: Set up Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ${{ matrix.xcode }}
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          FORMULAE=(boost openssl pkg-config)
+      
+          # GitHub Actions runners have a large number of pre-installed
+          # Homebrew packages. Don't waste time upgrading all of them.
+          brew list --versions "${FORMULAE[@]}" || brew update
+          for FORMULA in "${FORMULAE[@]}"; do
+            brew list --versions "$FORMULA" || brew install "$FORMULA"
+          done
+          # Ensure /usr/local/opt/openssl exists
+          brew reinstall openssl
+
+      - name: Configure ndn-cxx
+        run: |
+          cd ..
+          git clone https://github.com/named-data/ndn-cxx
+          cd ndn-cxx
+          git checkout ndn-cxx-0.8.0
+          ./waf configure
+          ./waf
+          sudo ./waf install
+          cd ../NAC-ABE
+      - name: Configure NFD
+        run: |
+          cd ..
+          git clone https://github.com/named-data/NFD
+          cd NFD
+          git checkout NFD-22.02
+          git submodule update --init
+          ./waf configure
+          ./waf
+          sudo ./waf install
+          sudo cp /usr/local/etc/ndn/nfd.conf.sample /usr/local/etc/ndn/nfd.conf
+          cd ../NAC-ABE
+          nfd-start
+
+      - name: Install OpenABE
+        run: |
+          cd ..
+          git clone https://github.com/zeutro/openabe.git
+          cd openabe
+          sudo -E ./deps/install_pkgs.sh
+          . ./env
+  
+          cd deps/relic
+          make
+          cd ../..
+  
+          NO_DEPS=1 BISON=$(which bison) make
+          make test
+          sudo -E make install
+          cd ../NAC-ABE
+
+      - name: Configure CMake
+        # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+        # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+        run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DHAVE_TESTS=True -DBUILD_EXAMPLES=True
+
+      - name: Build
+        # Build your program with the given configuration
+        run: cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
+
+      - name: Start NFD
+        run: sudo systemctl start nfd.service
```

Comment:
macOS doesn't use systemd (and nfd is already started with `nfd-start` in an earlier step)

Reply:
OK Thanks.